.codecov.yml

```diff
@@ -27,6 +27,7 @@ coverage:
   ignore:
     - "src/.*/test/.*"
     - "src/FortranChecks/f90sub/Draco_Test.F90"
+    - "src/mesh/*.in"
```

Comment:
@kgbudge I think you missed the `test` subdirectory: `src/mesh/test/*.in`. Sorry about not catching this earlier.